cartography/intel/aws/inspector.py

```diff
@@ -82,6 +83,10 @@ def get_inspector_findings(
     only fetch those in ACTIVE or SUPPRESSED statuses.
     Run the query in batches of 1000 findings and return an iterator to fetch the results.
     """
+    # Note: We can't use @aws_handle_regions decorator here because this function returns a generator.
```

Comment:
Making sure I understand: so right now, aws_handle_regions is not doing anything in aws inspector?

Reply:
Correct